alfasim_sdk/models.py

```diff
@@ -133,12 +133,12 @@ def data_model(*, caption: str, icon: Optional[str] = None) -> Callable:
 
         @data_model(icon='', caption='My Plugin')
         class MyModel:
-	        distance = Quantity(value=1, unit='m', caption='Distance')
+                distance = Quantity(value=1, unit='m', caption='Distance')
```

Comment:
Just curious, why did this extra indentation show up?

Reply:
Must be something related to the update pre-commit hooks, we were using black18.6b4, and since I updated to  20.8b1

Reply:
Highly doubt that: that's an internal string, black doesn't know that that triple quoted string contains Python ;)

Most likely it was your library that converted CamelCase to camel_case that messed up a bit.

Reply:
I didn't see the context, this is the docstring 😅